lib/utils/cli.go

```diff
@@ -294,6 +294,38 @@ func InitCLIParser(appName, appHelp string) (app *kingpin.Application) {
 	return app.UsageTemplate(defaultUsageTemplate)
 }
 
+// UpdateAppUsageTemplate updates usage template for kingpin application.
```

Comment:
Both comment and function name are too generic, maybe rename and mention what the function fixes.